core/c/libc/signal.odin

```diff
@@ -58,4 +45,5 @@ when ODIN_OS == .Darwin {
 	SIGINT   :: 2
 	SIGSEGV  :: 11
 	SIGTERM  :: 15
+	SIGTRAP  :: 5
```

Comment:
If this is added here, it would also need to be there for Windows, with the `libc` package I believe we want every symbol to be available on all targets.

If windows does not have a `SIGTRAP`, just defining it inline in the test signal handler is probably good too?

Reply:
Oh just saw your edit ;)

Reply:
I did some searching, and it looks like Windows doesn't have one. I'll figure out a fix.